Sources/SentrySwiftUI/SentryTracedView.swift

```diff
@@ -34,41 +34,61 @@ import SentryInternal
 ///         //The part of your content you want to measure
 ///     }.sentryTrace("My Awesome Screen")
 ///
-///
 @available(iOS 13, macOS 10.15, tvOS 13, watchOS 6.0, *)
 public struct SentryTracedView<Content: View>: View {
-    
+    @State var viewWasPresented = false
```

Comment:
`m`: I would stick to the SwiftUI domain language and call this 
```suggestion
    @State var viewAppeared = false
```